Build fails with 0.4.1

After #124 being fixed, the build now fails with:

```
$ tsc       
node_modules/@oruga-ui/theme-bulma/dist/types/bulma.d.ts:1:35 - error TS2307: Cannot find module '@oruga-ui/oruga-next' or its corresponding type declarations.

1 import type { OrugaOptions } from "@oruga-ui/oruga-next";
                                    ~~~~~~~~~~~~~~~~~~~~~~


Found 1 error in node_modules/@oruga-ui/theme-bulma/dist/types/bulma.d.ts:1
```

But `@oruga-ui/oruga-next` is not defined as even a peer dependency. If I add it as a dev dep, it then fails because it can't find `vue` and `vue-component-type-helpers`, and if I add those two as dev deps, it finally builds.

My code is pretty simple, it's a generic module that I use to "patch" the config for all my projects:

```typescript
import { bulmaConfig as originalBulmaConfig } from '@oruga-ui/theme-bulma';
import { merge } from 'lodash';

export const bulmaConfig = merge(
  originalBulmaConfig,

  {
    iconPack: 'fa',
    iconComponent: 'FontAwesomeIcon',
    field: {
      fieldInputWrapperClass: 'field-body',
    },
  },
);
```

I am unsure what is the correct fix for that use case.

--- a/package.json
+++ b/package.json
@@ -31,6 +31,9 @@
     "lint:style": "stylelint **/*.{css,scss} --fix --ignore-path .gitignore",
     "update": "ncu -u"
   },
+  "peerDependencies": {
+    "@oruga-ui/oruga-next": "0.9.0-pre.1"
+  },
   "dependencies": {
     "bulma": "1.0.2"
   },

